ArcGISRuntimeSDKQt_QMLSamples/UtilityNetwork/CreateLoadReport/CreateLoadReport.qml

```diff
@@ -0,0 +1,468 @@
+        traceConfig.functions.clear();
+        traceConfig.functions.append(addLoadAttributeFunction);
+
+        // Set to false to ensure that service points with incorrect phasing
+        // (which therefore act as barriers) are not counted with results.
+        traceConfig.includeBarriers = false;
+        return traceConfig;
+    }
+
+    function createCodedValuesPhaseList() {
+        phasesCurrentAttr = utilityNetwork.definition.networkAttribute(phasesNetworkAttributeName);
+
+        if (phasesCurrentAttr.domain.domainType === Enums.DomainTypeCodedValueDomain) {
+            return phasesCurrentAttr.domain.codedValues;
+        }
+    }
+
+    function runReport(selectedPhases) {
+        for (let i = 0; i < phaseCodedValuesList.length; i++) {
+            if (selectedPhases.includes(phaseCodedValuesList[i].name)) {
+                phaseQueue.push(phaseCodedValuesList[i]);
+            }
+        }
+
+        if (phaseQueue.length > 0) {
+            sampleStatus = CreateLoadReport.SampleStatus.Busy;
+            currentPhase = phaseQueue.pop();
+            createReportForPhase(currentPhase);
+        }
+    }
+
+    function createReportForPhase(phase) {
+        const condExpr = ArcGISRuntimeEnvironment.createObject("UtilityNetworkAttributeComparison", {
+                                                                 networkAttribute: phasesCurrentAttr,
+                                                                 comparisonOperator: Enums.UtilityAttributeComparisonOperatorDoesNotIncludeAny,
+                                                                 value: phase.code
+                                                             });
+
+        const traceOrCondition = ArcGISRuntimeEnvironment.createObject("UtilityTraceOrCondition", {
+                                                                     leftExpression: baseCondition,
+                                                                     rightExpression: condExpr
+                                                                 });
+
+        traceParams.traceConfiguration.traversability.barriers = traceOrCondition;
+        utilityNetwork.trace(traceParams);
+    }
+
+    // Load Report UI
+
+    Rectangle {
+        id: rectangle
+        anchors.horizontalCenter: parent.horizontalCenter
+        width: grid.width
+        height: contents.height
+
+        Column {
+            id: contents
+            anchors.fill: parent
+            padding: 10
+            spacing: 25
+
+            Row {
+                ButtonGroup {
+                    id: checkBoxes
+                    exclusive: false
+                    checkState: parentBox.checkState
+                }
+
+                GridLayout {
+                    id: grid
+                    columns: 4
+                    rowSpacing: 5
+
+                    CheckBox { id: parentBox; checkState: checkBoxes.checkState }
```

Comment:
seems like the same approach used in the C++ version with the repeater should be used here